.github/issue_template.md

```diff
@@ -1,7 +1,7 @@
 ### Prerequisites
 
 - [ ] Are you running the latest version of our extension?
-- [ ] Have you reviewed our [integration guide](https://www.taxjar.com/guides/integrations/magento2/)?
+- [ ] Have you reviewed our [integration guide](https://developers.taxjar.com/integrations/quides/magento2/)?
```

Comment:
Should the link here be 'https://developers.taxjar.com/integrations/guides/magento2/'?

Reply:
🤦‍♂️ 👍